src/vs/workbench/contrib/chat/browser/media/chat.css

```diff
@@ -979,6 +980,21 @@ have to be updated for changes to the rules above, or to support more deeply nes
 	border-style: dashed;
 	opacity: 0.75;
 }
+.chat-attached-context .chat-prompt-instructions-attachment .monaco-button {
+	border-left: 1px solid var(--vscode-chat-requestBorder, var(--vscode-input-background, transparent));
+	margin-left: 3px;
+}
+.chat-attached-context .chat-prompt-instructions-attachment.error .monaco-button,
+.chat-attached-context .chat-prompt-instructions-attachment.warning .monaco-button {
+	border-left-color: currentColor;
+}
+.chat-attached-context .chat-prompt-instructions-attachment:focus .monaco-button {
+	outline: none;
+	border-color: var(--vscode-focusBorder);
+}
```

Comment:
Why do you need this?

Reply:
Otherwise the [border](https://github.com/microsoft/vscode/pull/239514/files/9cc20b321a00de927fede6149aeec20517250ecd#diff-d9537b4703002189d5579da7ffbdd4c0d457f8cebea706f8c0e56cf4861dac44R984) stays the same color while all others change to blue. That is the same logic we have for existing borders extended to this one.

Reply:
I would have expected this to be handled by some general `.monaco-button:focus` rule? If you're copying this from elsewhere all good, but it looks like a potential code smell to me.

Reply:
Yeah I'd like it to be general component that I could reuse, but its not the case rn 🤔

Having a style that will allow this would in the monaco styles themself would be redundant and might feel like a code smell on its own - these styles are not for the monaco-buttom focus, but focus of the `parent`. Hence such styles would be something like `*:focus .monaco-button {}` which would not only have a negative performance impact, but also would require these styles to be reset for the components that don't need them 🤔 

We also have many similar styles across the codebase (and many codesmells indeed!), including changes to the styles of its border (e.g., [here](https://github.com/microsoft/vscode/blob/main/src/vs/base/browser/ui/dialog/dialog.css#L154)) 🐰